tools/ipapy2/macros.xml

```diff
@@ -1,3 +1,27 @@
 <macros>
     <token name="@TOOL_VERSION@">1.3.0</token>
+
+    <xml name="citations">
+        <citations>
+        <citation type="doi">10.1021/acs.analchem.9b02354</citation>
+        <citation type="doi">10.1093/bioinformatics/btad455</citation>
+        </citations>
+    </xml>
+
+    <xml name="ionisation">
+        <param label="ionisation" name="ionisation" type="select" help="ionisation. positive +1 or negative -1">
+            <option value="positive">positive</option>
+            <option value="negative">negative</option>
+        </param>
+    </xml>
+
+    <xml name="ncores">
+        <param label="number of cores" name="ncores" type="integer" value="1" help="number of cores">
+        </param>
+    </xml>
+
+<token name="@init_logger@">
+import pandas as pd
+from ipaPy2 import ipa
+</token> 
```

Comment:
This macro should probably be renamed as it doesn't init the logger but actually does the imports.